packages/app-spec-squad-chess/src/Matchmaker.js

```diff
@@ -0,0 +1,246 @@
+import m from 'mithril'
+import { p2p } from '@squad/sdk'
+import crypto from 'crypto'
+import settings from './settings.json'
+import state from './state.js'
+import { checkWinner } from './Board.js'
+
+export const Matchmaker = {
+  oninit: () => {
+    state.p2p.id = crypto.randomBytes(16).toString('hex')
+    console.log(`Our matchmaking Id: ${state.p2p.id}`)
+
+    state.p2p.room = 'Room name'
```

Comment:
Why is this initialized to 'Room name'?

Reply:
It was getting prefilled from here into the form, but I don't think there's a reason to do it that way, changing it.